src/test/java/core/basesyntax/service/read/FileReaderImplTest.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.read;
+
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileReaderImplTest {
```

Comment:
lets add some negative tests (tests with null of invalid file path). Same in FileWriter